packages/ckeditor5-image/src/imageupload/utils.js

```diff
@@ -68,7 +69,7 @@ export function fetchLocalImage( image ) {
  * @returns {Boolean}
  */
 export function isLocalImage( node ) {
-	if ( !node.is( 'element', 'img' ) || !node.getAttribute( 'src' ) ) {
```

Comment:
I'm not sure if using this util makes sense here. Especially if it would return true only for `img` that is not inside the `figure`.

Reply:
This is related to my response in [https://github.com/ckeditor/ckeditor5/pull/9351#discussion_r605569183.](https://github.com/ckeditor/ckeditor5/pull/9351#discussion_r605569183.)